pom.xml

```diff
@@ -271,18 +279,11 @@
       <scope>provided</scope>
     </dependency>
 
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <version>${guava.version}</version>
-    </dependency>
-
```

Comment:
@Vipinofficial11 please fix

Reply:
This dependency is actually just shifted from bottom -> top because we needed the guava version 27.0.1 but some other dependencies were using the lower versions so to override them had to move it at top.